src/main/java/org/opensearch/security/dlic/rest/api/AbstractApiAction.java

```diff
@@ -109,20 +109,21 @@ protected void handleApiRequest(final RestChannel channel, final RestRequest req
 		try {
 			// validate additional settings, if any
 			AbstractConfigurationValidator validator = getValidator(request, request.content());
-			if (!validator.validate()) {
+			if (validator!= null && !validator.validate()) {
```

Comment:
I see that this PR introduces a validator. When will the validator ever be null? Are the changes in this file needed?

Reply:
This was added while testing and I forgot to remove it. Thanks @cwperks  for pointing this out. Will remove this check condition in next commit.